test-suite/Makefile

```diff
@@ -146,6 +140,7 @@ VSUBSYSTEMS := prerequisite success failure bugs bugs-nocoqchk output output-coq
 # All subsystems
 SUBSYSTEMS := $(VSUBSYSTEMS) misc ide ide vio coqchk output-coqchk coqwc coq-makefile tools $(UNIT_TESTS)
 
+# EJGA: This seems dangerous as first target...
```

Comment:
we should just put `.DEFAULT_GOAL := all` somewhere (end or beginning of file) as we do in CoqMakefile.in

Reply:
I did try that in the past, however I was not able to make it work lol